libs/community/langchain_community/chat_models/zhipuai.py

```diff
@@ -133,19 +151,22 @@ def _convert_delta_to_message_chunk(
     role = dct.get("role")
     content = dct.get("content", "")
     additional_kwargs = {}
-    tool_calls = dct.get("tool_call", None)
+    tool_calls = dct.get("tool_calls", None)
```

Comment:
was `dct.get("tool_call", None)` a bug earlier?

Reply:
Yes, when the return is a tool call, zhipuai use "tool_calls" as the key